approval_to_apb.rb

```diff
@@ -0,0 +1,50 @@
+class ApprovalToAPB
+
+  def initialize(options)
+    @bpms_url       = options[:bpms_url]
+    @bpms_username  = options[:bpms_username]
+    @bpms_password  = options[:bpms_password]
+    @role_repo      = options[:role_repo]
+    @source_dir     = File.dirname(__FILE__)
+    @apb_dir        = "."
+  end
+
+  def set_defaults
+    template_file = File.join(@source_dir, "/templates/approval_role_defaults.yml")
+    raise "templates/approval_role_defaults.yml not found in #{@source_dir}" unless File.exist?(template_file)
+    role_defaults = YAML.load_file(template_file)
+    role_defaults['rules_engine_base_url'] = @bpms_url
```

Comment:
@syncrou Wouldn't these parameters be accessible in the ansible yml directly with the {{ rules_engine_base_url }}